controller/scheduler/swarm.py

```diff
@@ -32,11 +30,15 @@ def create(self, name, image, command='', template=None, **kwargs):
             if mem[-2:-1].isalpha() and mem[-1].isalpha():
                 mem = mem[:-1]
         cpu = kwargs.get('cpu', {}).get(l['c_type'])
-        self.docker_cli.create_container(image=cimage, name=name,
-                                         command=command.encode('utf-8'), mem_limit=mem,
-                                         cpu_shares=cpu,
-                                         environment=[affinity],
-                                         host_config={'PublishAllPorts': True})
+        try:
+            self.docker_cli.create_container(image=cimage, name=name,
+                                             command=command.encode('utf-8'),
+                                             mem_limit=mem,
+                                             cpu_shares=cpu,
+                                             environment=[affinity],
+                                             host_config={'PublishAllPorts': True})
+        except errors.APIError as err:
+            raise errors.DockerException(str(err.message)+"  "+str(err.explanation))
```

Comment:
I think it's better that we don't add a try/catch block for this issue. [`.create()`](https://github.com/deis/deis/blob/68415ae735c46131e48f818eb049da6bad357fd0/controller/api/models.py#L492-L495) will catch the original error and report it to the user, so we don't need to do any try/catch magic here.